packages/list-exports/index.js

```diff
@@ -274,6 +277,7 @@ module.exports = async function listExports(packageJSON) {
 					packageDir,
 					process: processPreExportsFile,
 					addError,
+					skipSlashExport: true,
```

Comment:
what i like about this approach is that it still gives a way to restore it, for nodes v13.2-v13.12, when I add tests in the future that verify that all of the found exports are indeed requireable/importable.